Documentation/filters-strings.md

```diff
@@ -166,7 +166,9 @@ This filter accepts a parameter (boolean, default `false`) that controls the pre
 
 ## Filter: `swiftIdentifier`
 
-Transforms an arbitrary string into a valid Swift identifier (using only valid characters for a Swift identifier as defined in the Swift language reference). It will apply the following rules:
+This filter has a couple of modes that you can specifiy using an optional argument (defaults to "normal"):
+
+**normal**: Transforms an arbitrary string into a valid Swift identifier (using only valid characters for a Swift identifier as defined in the Swift language reference). It will apply the following rules:
 
 - Uppercase the first character.
```

Comment:
The thing is, `snakeToCamelCase` would uppercase the first letter anyway, no way around that.

And I'm not talking about swift 2, but even swift 3/4 types need an uppercase first letter. Only variables and cases don't.